esite/settings/base.py

```diff
@@ -2,173 +2,268 @@
 Django base settings for esite project.
 
 For more information on this file, see
-https://docs.djangoproject.com/en/2.2/topics/settings/
+https://docs.djangoproject.com/en/stable/topics/settings/
 
 For the full list of settings and their values, see
-https://docs.djangoproject.com/en/2.2/ref/settings/
+https://docs.djangoproject.com/en/stable/ref/settings/
 """
 
 import os
 
 
```

Comment:
The env variable has to be assigned in the base settings.
```suggestion
env = os.environ.copy()

```